wp-content/themes/wp-starter/blocks/video-embed/render.twig

```diff
@@ -20,5 +20,5 @@
 	allowedBlocks: allowed
 } %}
 <section {{ block_attrs( attributes ) }}>
-  {{ inner_blocks( inner ) }}
+  <div class="flex flex-col">{{ inner_blocks( inner ) }}</div>
```

Comment:
For custom blocks in WordPress, we have the ability to set the default layout in the `block.json` file, which we have configured to be flex-box columns. We also have the flexibility to change this default as needed (applies to both `php` and `twig` files), and even support the UI controls that allow the user to change from col to row, if we wanted to (I understand that's unlikely). But I'm curious why we are wrapping the inner blocks in another `<div>` with the same styles, but with less control/capabilities. If the styles applied to the `<section>` are wrong, we can certainly adjust them to be correct.

Reply:
Honestly it's probably leftover from when I was trying to align the layout classes with Blueprint. I was going to use `gap` to adding spacing between heading/description/video, but ultimately that wasn't going to work. I removed the wrapper around `innerBlocks` and it works fine as-is.